recipes/napari-3d-counter/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "napari-3d-counter" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-3d-counter-{{ version }}.tar.gz
+  sha256: 06e12f1f0837266e1b190e0d65b714202b2005303afc76e7f2f9b478d22f0c1d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools >=42.0.0
+    - setuptools-scm
+    - wheel
+  run:
+    - magicgui
+    - matplotlib-base
+    - napari
+    - numpy
+    - python >=3.8
+    - qtpy
+
+test:
+  imports:
+    - napari_3d_counter
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+    - pyqt
```

Comment:
Same issue as in the other qt depending recipe.

Reply:
Napari by default does not provide a binding also on pip, so a user needs to install napari and then a binding, pyside or pyqt, hence why all these plugins will need to include a binding so the imports or tests pass.

This is not something we need to fix. Is a packaging decision. See https://napari.org/stable/tutorials/fundamentals/installation.html

Reply:
The variant option already discussed and voted against.

See:
- https://github.com/conda-forge/napari-feedstock/pull/48
- https://github.com/conda-forge/napari-feedstock/issues/47

The napari team is ok with having this as is for both conda and pip.

Reply:
Fine with me then.